packages/blade/src/components/Accordion/_decisions/decisions.md

```diff
@@ -0,0 +1,159 @@
+# Accordion
+
+An accordion is used to allow users to toggle between different content sections in a compact vertical stack.
+
+This document outlines the API of `Accordion` component.
+
+<div align="center">
+  <img src="./accordion-thumbnail.png" width="380" alt="" />
+</div>
+
+## Design
+
+- [Figma - Accordion](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=79-619941&t=Rcrabprdzrz7JdId-0)
+
+## Interactions
+
+## API
+
+Annotated component with props ([figma](https://www.figma.com/file/wzCTQ9kMjWWHKFu7kCzYGi/Accordion-breakdown?type=design&node-id=0-1&t=GfrkTz89y2M46wQN-0)):
+
+![Accordion breakdown](./accordion-breakdown.png)
+
+Sample usage, composes `Accordion` and `AccordionItem`:
+
+```jsx
+import { Accordion, AccordionItem } from '@razorpay/blade';
```

Comment:
On design I see we are using `AccordionGroup` and `AccordionItem` can we keep both in sync? my vote will be `Accordion` and `AccordionItem` itself